Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3339,3 +3458,114 @@ def get_dynamic_analysis_command(client: CoreClient, args: Dict) -> CommandResul
         outputs=filtered_alerts,
         raw_response=raw_response,
     )
+
+
+def add_tag_to_endpoints_command(client: CoreClient, args: Dict):
+    endpoint_ids = argToList(args.get('endpoint_ids', []))
+    tag = args.get('tag')
+
+    if any(  # check if any filter arguments provided
```

Comment:
intersect might be a bit more straightforward. Whichever you think is more clear
```
if args & {
            'endpoint_id_list', 'dist_name', 'ip_list', 'group_name', 'platform', 'alias_name',
            'isolate', 'hostname', 'status', 'first_seen_gte', 'first_seen_lte', 'last_seen_gte', 'last_seen_lte'
        }:
```

Reply:
I like your way, but since `args` is a dict, it should be `set(args.keys())`, will change it.